wdl/ConcatTextFiles.wdl

```diff
@@ -1,43 +0,0 @@
-version 1.0
-
-import "TasksMakeCohortVcf.wdl" as tasks
```

Comment:
What's the plan for utility workflows like this that we may want to use again, but rely on WDLs in the gatk-sv repo that haven't been moved to gatk-sv-internal? ie. here ConcatTextFiles needs TasksMakeCohortVcf.wdl - but we may not want to maintain two versions

Reply:
Good point, I will copy those dependencies over into gatk-sv-internal since it's possible that they could get desynced with the dependent WDLs over time. I'm not worried about having a little redundant code over there if they are just a few core utility tasks.